spec/services/manage_action_spec.rb

```diff
@@ -74,12 +79,10 @@
       end
 
       it 'saves all new fields to action_user' do
```

Comment:
yeah it keys off email, so this spec was actually creating a new record and not at all doing what it purported to be doing.